src/app/filecoin-plus/allocators/data/allocatorList.ts

```diff
@@ -0,0 +1,21 @@
+type AllocatorType = 'Automatic' | 'Manual' | 'Market-Based'
```

Comment:
```suggestion
type AllocationMethod = 'Automatic' | 'Manual' | 'Market-Based'
```

Reply:
Ah sorry, forgot to delete. This file is redundant